docs/user/commercial/index.rst

```diff
@@ -1,10 +1,41 @@
-Read the Docs for Business
---------------------------
+About |com_brand|
+=================
 
-Read the Docs is our community solution for open source projects at `readthedocs.org`_
-and we offer Read the Docs for Business
-for building and hosting commercial documentation at `readthedocs.com`_.
-Features in this section are specific to Read the Docs for Business.
+.. this page is currently moving towards becoming "About Read the Docs for Business"
+.. rather than an index of features.
+
+We offer *|com_brand|*
+for building and hosting commercial documentation.
+
+Our commercial solutions are provided as a set of subscriptions that are paid and managed through an online interface.
+In order to get started quickly and easily, a `trial option <https://about.readthedocs.com/pricing/>`__ is provided free of charge.
+
+.. seealso::
+
+   `Read the Docs website: Features <https://about.readthedocs.com/about/>`__
+      A general presentation of our features is available on our website, and the
+      `pricing page <https://about.readthedocs.com/pricing/>`__ lists all details and features for each subscription level.
+
+   `Read the Docs website: Company presentation <https://about.readthedocs.com/company/>`__
+      Information about the company running Read the Docs, including our mission, team, and community.
+
+
+Commercial documentation solutions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to providing the same features as |org_brand|,
+commercial subscriptions to Read the Docs add additional features based on the same software platform but on a separate infrastructure.
+
+.. figure:: /img/community_in_business.png
+   :align: center
+   :scale: 50%
+
+   Read the Docs for Community and Business are a combined system:
+   All features developed for community benefit the business solution, and a lot of solutions developed for business
+   are implemented for community.
```

Comment:
```suggestion
   are implemented for the community.
```

I feel like using `business` and `community` as shorthand doesn't really work.

Reply:
no, you're right!